examples/3DTilesOutliers.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>3D Tiles Outliers example</title>
```

Comment:
Rename example file 3DTilesWireframe + html title and add a mosaic image in index.html

Reply:
done